examples/verifying_gfd/verifying_gfd1.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import desbordante
+
+class bcolors:
+    HEADER = '\033[95m'
+    WARNING = '\033[93m'
+    ENDC = '\033[0m'
+
+graph_name = 'blogs_graph'
+modified_graph_name = 'incorrect_author_blogs_graph'
+gfd_name = 'correct_author_gfd'
+
+GRAPHS_DATASETS_FOLDER_PATH = 'examples/verifying_gfd/datasets/graphs'
+GFDS_DATASETS_FOLDER_PATH = 'examples/verifying_gfd/datasets/gfds'
+
+GRAPH = Path(f'{GRAPHS_DATASETS_FOLDER_PATH}/{graph_name}.dot')
+MODIFIED_GRAPH = Path(f'{GRAPHS_DATASETS_FOLDER_PATH}/{modified_graph_name}.dot')
+GFD = Path(f'{GFDS_DATASETS_FOLDER_PATH}/{gfd_name}.dot')
+
+GRAPHS_FIGURES_FOLDER_PATH = 'examples/verifying_gfd/figures/graphs'
+GFDS_FIGURES_FOLDER_PATH = 'examples/verifying_gfd/figures/gfds'
+
+GRAPH_IMAGE = Path(f'{GRAPHS_FIGURES_FOLDER_PATH}/{graph_name}.png')
+MODIFIED_GRAPH_IMAGE = Path(f'{GRAPHS_FIGURES_FOLDER_PATH}/{modified_graph_name}.png')
+GFD_IMAGE = Path(f'{GFDS_FIGURES_FOLDER_PATH}/{gfd_name}.png')
+
+GRAPH_INFO = 'The graph is depicted in figure. The following abbreviations were used: A - account, B - blog. '\
```

Comment:
Wrap in parentheses instead of using backslashes.
https://peps.python.org/pep-0008/#maximum-line-length

Reply:
Same for the other files